src/cli/java/org/commcare/util/screen/EntityScreen.java

```diff
@@ -318,4 +318,18 @@ public String toString() {
     public Hashtable<String, TreeReference> getReferenceMap() {
         return referenceMap;
     }
+
+    @SupressWarnings("unsused")
+    public boolean referencesContainStep(String stepValue) {
```

Comment:
This is going to be a fairly expensive call, by the way, since it re-processes all of the datum outputs. Is this being done for performance reasons?